cfgmgr/intfmgr.cpp

```diff
@@ -449,6 +451,31 @@ bool IntfMgr::isIntfStateOk(const string &alias)
     return false;
 }
 
+void IntfMgr::delIpv6LinkLocalNeigh(const string &alias)
+{
+    vector<string> neighEntries;
+
+    SWSS_LOG_INFO("Deleting ipv6 link local neighbors for %s", alias.c_str());
+
+    m_neighTable.getKeys(neighEntries);
+    for (auto neighKey : neighEntries)
+    {
+        if (!neighKey.compare(0, alias.size(), alias.c_str()))
+        {
+            vector<string> keys = tokenize(neighKey, ':', 1);
+            if (keys.size() == 2)
+            {
+                IpAddress ipAddress(keys[1]);
+                if ((ipAddress.isV4() == false) && (ipAddress.getAddrScope() == IpAddress::AddrScope::LINK_SCOPE))
```

Comment:
Won't this isV4 check be a problem if the changes mentioned in these PR are merged? https://github.com/Azure/sonic-swss/pull/1903